README.md

```diff
@@ -186,3 +186,23 @@ Due to the way we inject the thrashers into `frontend` we can't add `<script>` t
 ```
 
 Although the same practice applies to javascript as it does for animations and transitions. There has to be a good reason for it to exist and they have to be incredibly light. It's also best not to require heavy libraries like `jQuery` to achieve the same results that could be done with vanilla javascript and/or micro-libraries.
+
+
+### Testing Updates
```

Comment:
If there is anything I've missed out, or you have any questions, I am happy to add to this README.